### PR TITLE
TFW-2886 - Modify page breaks for printing Interaction

### PR DESCRIPTION
**Pull request by jasonhale**
When printing an interaction, the timeline started on a new page, instead of below the header info.  Just took out an extraneous page-break.

Print out after changes:
[RCI components _ Interaction Editor - View Mode.pdf](https://github.com/topbox-rci/rci-components/files/2886634/RCI.components._.Interaction.Editor.-.View.Mode.pdf)

A pdf illustrating the issue is [attached to the ticket](https://topbox.atlassian.net/browse/TFW-2886).

### CODE REVIEW

**Comment by jasonhale** [comment]
I stopped short of that change because the interaction viewer has been temperamental lately and we have some redesign/updates on the near horizon.

But, yeah, I can make those updates. No prob.

**Comment by jasonhale** [comment]
@troyblank, BEMification complete... at the top level, at least.

File: components/interactionEditor/interactionEditor.beta.scss [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    height: 100%;
+
+    @media print {
+        display: block;
+        height: auto;
+    }
+}
+
+.interaction-editor__timeline,
+.interaction-editor__actionpane {
```
Comment (troyblank) [inline components/interactionEditor/interactionEditor.beta.scss]: This could be moved to it's own scss file since it has it's own component.

Comment (jasonhale) [reply components/interactionEditor/interactionEditor.beta.scss]: Those are the wrapping `div`s for TimelineScroller and ActionpaneScroller.  Can't move those at the moment.

Comment (troyblank) [reply components/interactionEditor/interactionEditor.beta.scss]: oops sorry I wrote this in the wrong spot, I meant for it to be at `.scroller--actionpane`

File: components/interactionEditor/interactionEditor.beta.scss [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    @media print {
+        display: block;
+        width: 100%;
+        height: auto;
+        margin: 0;
+        overflow: visible;
+    }
+
+    .scroller--timeline,
+    .scroller--actionpane {
```
Comment (troyblank) [inline components/interactionEditor/interactionEditor.beta.scss]: This could be moved to it's own scss file since it has it's own component.

Comment (jasonhale) [reply components/interactionEditor/interactionEditor.beta.scss]: ha!  i'm in the process of doing that right now.

File: components/interactionEditor/scroll/scroll.beta.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
+@import '../../../assets/sass/base/base';
+
+.scroll {
```
Comment (troyblank) [inline components/interactionEditor/scroll/scroll.beta.scss]: This name is pretty generic, do you think it will conflict?

Comment (jasonhale) [reply components/interactionEditor/scroll/scroll.beta.scss]: It is.  And may.  Had that concern, too.  "Scroller" a better, non-generic name?

**Pull request merged**